app/src/main/res/values/themes.xml

```diff
@@ -11,7 +11,7 @@
     <!-- Must be included for dialog styling -->
```

Comment:
Ignore location of this comment

We normally don't write the ticket ID in the commit title but add it to a separate line of the commit description